challenges/how-to-create-a-list.py

```diff
@@ -0,0 +1,31 @@
+# The challenge is to write a script that can generate all the fibonacci numbers form 1 to 100
+
+def using_normal_loops():
+    fibonacci = [1, 1]
+    for i in range(2, 100):
+        previous_number_1 = fibonacci[i - 1]
+        previous_number_2 = fibonacci[i - 2]
+        new_number = previous_number_1 + previous_number_2
+        fibonacci.append(new_number)
+
+    for idx, val in enumerate(fibonacci):
```

Comment:
Very cool! I think this function should probably just return the list (if we're following the challenge strictly). That said, this looks great.

Reply:
Sure, why not?